cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/CgmesMetadataModel.java

```diff
@@ -166,4 +166,12 @@ public interface CgmesMetadataModel {
      * @return The model with an empty set of values this model depends on.
      */
     CgmesMetadataModelImpl clearDependencies();
+
+    /**
+     * Remove all the model ids this model supersedes.
+     * @return The model with an empty set of values this model supersedes.
+     */
+    default CgmesMetadataModelImpl clearSupersedes() {
```

Comment:
Why do you need a default method here?

Reply:
Because the interface was already published in a previous version, and we are extending it. If other implementations have been created, they will fail to compile if we do not add a default.

Reply:
The patch release has other breaking changes, so after talking with @olperr1 we prefer the default behavior to be removed.